sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -229,6 +229,7 @@ public partial class RequestFailedException : System.Exception, System.Runtime.S
         public string? ErrorCode { get { throw null; } }
         public int Status { get { throw null; } }
         public override void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public Azure.Response? GetRawResponse() { throw null; }
```

Comment:
Nit: is this just so it matches our `Response<T>` implementation? If so, cool; otherwise, all other languages just have a `Response` property (cased idiomatically, of course) that might be good to consider for simplicity.

Reply:
Yes, see this conversation - https://github.com/Azure/azure-sdk-for-net/pull/35716#discussion_r1174020856